cmd/containerd-shim-runhcs-v1/stats/stats.proto

```diff
@@ -31,4 +55,16 @@ message VirtualMachineProcessorStatistics {
 
 message VirtualMachineMemoryStatistics {
 	uint64 working_set_bytes = 1;
+	int32 virtual_node_count = 2;
```

Comment:
I guess as long as were at it: `uint8`

Reply:
Protobuild doesn't seem to like uint8. Should I change to uint32 or would you prefer this not be changed then?

Reply:
Lame. `uint32` it is. No byte type it looks like